requirements.txt

```diff
@@ -2,7 +2,7 @@ wheel>=0.23.0
 numpy>=1.10.2
 pymongo>=3.0
 matplotlib>=1.3.0
-scipy>=0.15
+scipy>=0.17
```

Comment:
I'll bump to 0.18, this has been out for a while, and we don't know if it works on stuff we don't use ourselves